<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>841</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Personnel</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_27">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>351</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#groupeBox

{
        content: &quot;&quot;;
        position: fixed;
        height: 100%; width: 100%;
        background: url(images/legumes.jpeg);
        background-size: cover;
        z-index: -1; /* Keep the background behind the content */     
        -webkit-filter: blur(8px);
    -webkit-background-size: cover; /* pour Chrome et Safari */
    -moz-background-size: cover; /* pour Firefox */
    -o-background-size: cover; /* pour Opera */
    background-size: cover;
}</string>
     </property>
     <property name="title">
      <string>AJOUTER UN PERSONNEL</string>
     </property>
     <widget class="QLabel" name="cin_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cin">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_65">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>NOM</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_67">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>PRENOM</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 255);</string>
      </property>
      <property name="text">
       <string>AJOUTER</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numtel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="cin_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>ADRESSE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Date_naissance</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>NUM_TEL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="adresse">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_naissance">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QTreeView" name="tabperso">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>601</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 228, 240);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_idch_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>100</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(253, 184, 255);
</string>
     </property>
     <property name="text">
      <string>CHERCHER  PAR NOM</string>
     </property>
    </widget>
    <widget class="QPushButton" name="QRcodecom">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>450</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Generer QR code</string>
     </property>
    </widget>
    <widget class="QLabel" name="qrcodecommande">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>580</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="qrlabel">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>400</y>
       <width>251</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
 border-width: 1px;
border-style: solid;
border-color: black;
  border-radius: 10px;
padding:0 5;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>400</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#groupeBox

{
        content: &quot;&quot;;
        position: fixed;
        height: 100%; width: 100%;
        background: url(images/legumes.jpeg);
        background-size: cover;
        z-index: -1; /* Keep the background behind the content */     
        -webkit-filter: blur(8px);
    -webkit-background-size: cover; /* pour Chrome et Safari */
    -moz-background-size: cover; /* pour Firefox */
    -o-background-size: cover; /* pour Opera */
    background-size: cover;
}</string>
     </property>
     <property name="title">
      <string>SUPPRIMER</string>
     </property>
     <widget class="QPushButton" name="supp">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 166, 247);</string>
      </property>
      <property name="text">
       <string>SUPRIMER</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cinmod_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_90">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CIN :</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>400</y>
       <width>221</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#groupeBox

{
        content: &quot;&quot;;
        position: fixed;
        height: 100%; width: 100%;
        background: url(images/legumes.jpeg);
        background-size: cover;
        z-index: -1; /* Keep the background behind the content */     
        -webkit-filter: blur(8px);
    -webkit-background-size: cover; /* pour Chrome et Safari */
    -moz-background-size: cover; /* pour Firefox */
    -o-background-size: cover; /* pour Opera */
    background-size: cover;
}</string>
     </property>
     <property name="title">
      <string>MODIFIER</string>
     </property>
     <widget class="QPushButton" name="mod_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 85, 127);
background-color: rgb(255, 170, 255);</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Numtel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QLabel" name="cin_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="exel">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>40</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>excel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="CIN">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>83</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>83</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>270</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
border-radius: 10px;
padding: 0 8px;
background:#FFB6C1	;
font: bold, &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>210</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>on</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>210</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>off</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
